### PR TITLE
fix

### PR DESCRIPTION
**Pull request by istanya**
Changes in this pull request:

"Your first Django project!" section:
- changed settings ALLOWED_HOSTS

"Introduction to HTML" section:
- changed PythonAnywhere project path

"Extend your application" section: 
- changed PythonAnywhere project path
- added "warning_icon"

minor fixes

Edits are performed with @sudareva

### CODE REVIEW

**Comment by magul** [comment]
Hi, thank you for your fixes.

We tend to maintain translations on Crowdin (Russian is here: https://crowdin.com/project/django-girls-tutorial/ru#). Can you, please, apply these fixes also there?

Thanks in advance!

File: ru/python_introduction/README.md [hunk 1]
```diff
@@ -277,7 +279,7 @@ Maria
 
 Эта команда не возвращает ничего, она просто меняет порядок номеров в списке. Давайте выведем его на экран и посмотрим, что получилось:
 
-{% filename %}command-line{% endfilename %}
+{% filename %}}command-line{% endfilename %}
```
Comment (telezhnaya) [inline ru/python_introduction/README.md]: I guess this is redundant

Comment (telezhnaya) [inline ru/python_introduction/README.md]:
```suggestion
{% filename %}command-line{% endfilename %}
```

File: ru/django_urls/README.md [hunk 2]
```diff
@@ -49,7 +49,7 @@ URL-адрес раздела администрирования, который
 
 Мы также хотим сохранить файл `mysite/urls.py` в максимально аккуратном виде, так что мы импортируем URL-адреса для нашего приложения `blog` в `mysite/urls.py`.
 
-Вперёд, добавь строку для импорта `blog.urls`. Обрати внимение, что здесь мы используем функцию `include`, поэтому тебе придется импортировать её в первой строке своего файла.
+Вперёд, добавь строку для импорта blog.urls. Обрати внимание, что здесь мы используем функцию include, поэтому тебе придется импортировать её в строке from django.urls....
```
Comment (sudareva) [inline ru/django_urls/README.md]: Add extra change to this line

**Comment by sudareva** [comment]
> We tend to maintain translations on Crowdin (Russian is here: https://crowdin.com/project/django-girls-tutorial/ru#). Can you, please, apply these fixes also there?

@magul Hi! We added some fixes on crowdin, but most of them had already been there.

**Comment by istanya** [comment]
@magul this is a gentle reminder about our pull request. All changes are synced with crowdin. Our workshop at Kazan will take place tomorrow, so it would be cool to merge this today. Thank you!

**Comment by magul** [comment]
@istanya by default I merge a pull request after being approved by somebody from language-specific reviewer group. I this case we're still waiting for @DjangoGirls/ru-pr to approve.

Also, there are still comments from @telezhnaya which still we're not addressed (and even I'm not Russian language speaker I can see they are valid), so in such shape, this pull request cannot be merged.

**Comment by LenaVolzhina** [comment]
@magul thank you for the quick response! Is everything all right now?

**Comment by magul** [comment]
@LenaVolzhina, I'm not Russian language speaker, so I'm not able to review it properly. I can only say that @telezhnaya (technical) issues were not addressed so actually this pull request will cause Russian language version to be rendered with some kind of errors. I saw that you approved it, so do you want me to merge it in such shape?

**Comment by magul** [comment]
Ok, saw that @istanya resolves the issue pointed by @telezhnaya. So should I assume that's approved @LenaVolzhina and merge it?

**Comment by LenaVolzhina** [comment]
@magul Thank you so much!

File: ru/django_urls/README.md (hunk 2 above)
Comment (sudareva) [reply ru/django_urls/README.md]: Thank you for your carefulness!

**Comment by sudareva** [comment]
@magul @LenaVolzhina @istanya @b0g3r thanks to you all! You are amazing!

**Pull request merged**